src/textbooks/empty-placeholder/EmptyPlaceholder.test.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import { IntlProvider } from '@edx/frontend-platform/i18n';
+
+import EmptyPlaceholder from './EmptyPlaceholder';
+import messages from './messages';
+
+const onCreateNewTextbookMock = jest.fn();
+
+const renderComponent = () => render(
+  <IntlProvider locale="en">
+    <EmptyPlaceholder onCreateNewTextbook={onCreateNewTextbookMock} />
+  </IntlProvider>,
+);
+
+describe('<EmptyPlaceholder />', () => {
+  it('renders EmptyPlaceholder component correctly', () => {
+    const { getByText, getByRole } = renderComponent();
+
+    expect(getByText(messages.title.defaultMessage)).toBeInTheDocument();
+    expect(getByRole('button', { name: messages.button.defaultMessage })).toBeInTheDocument();
+  });
+
+  it('calls the onCreateNewTextbook function when the button is clicked', () => {
+    const { getByRole } = renderComponent();
+
+    const addButton = getByRole('button', { name: messages.button.defaultMessage });
+    fireEvent.click(addButton);
```

Comment:
[important]: We agreed to use `userEvent` in this project. 

```suggestion
    userEvent.click(addButton);
```

Reply:
Fixed